### PR TITLE
add persistence config

### PR DESCRIPTION
**Pull request by wcjohnson11**
:art: :cookie:
@sperand-io

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ describe('Mixpanel', function() {
     token: 'x',
     cookieName: 'y',
     crossSubdomainCookie: true,
-    secureCookie: true
+    secureCookie: true,
+    persistence: 'localStorage'
```
Comment (sperand-io) [inline test/index.test.js]: we should check that this worked with a specific `getConfig` check!

File: lib/index.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ var Mixpanel = module.exports = integration('Mixpanel')
   .option('cookieName', '')
   .option('crossSubdomainCookie', false)
   .option('secureCookie', false)
+  .option('persistence', 'cookie')
```
Comment (sperand-io) [inline lib/index.js]: will also need to add this to the comparison in the test

**Comment by sperand-io** [comment]
few comments inline!

File: test/index.test.js (hunk 1 above)
Comment (wcjohnson11) [inline test/index.test.js]: Looks like they removed get_config() from their mixpanel object...

I wrote this test and it passes but it's super brittle, should we include this test or leave as is
'''
      it('should switch to localStorage persistence if specified', function() {
        mixpanel.options.persistence = 'localStorage';
        analytics.initialize();
        analytics.page();
        analytics.called(mixpanel.load);
        analytics.assert(window.mixpanel._i[0][1].persistence === 'localStorage');
      });
'''

File: lib/index.js (hunk 2 above)
Comment (wcjohnson11) [inline lib/index.js]: added, let me know when it's g2g! @sperand-io

File: test/index.test.js [hunk 3]
```diff
@@ -42,6 +42,7 @@ describe('Mixpanel', function() {
       .option('people', false)
       .option('token', '')
       .option('trackAllPages', false)
+      .option('persistence', 'cookie')
       .option('trackNamedPages', true)
```
Comment (wcjohnson11) [inline test/index.test.js]: test added, let me know when it's g2g! @sperand-io

**Pull request merged**